src/abyssal_tome/constants.py

```
@@ -0,0 +1,99 @@
+from pathlib import Path
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)